pyproject.toml

```diff
@@ -47,6 +47,7 @@ dependencies = [
     "ophyd",
     "pysumreg",
     "qtpy",
+    "PyQt5>5.15",
```

Comment:
any version of pyqt5, pyqt6, pyside2, or pyside6 _should_ work (and if it does not it is a bug in bluesky).

Reply:
We're using PyQt5 in other places in our local work.  Is any part of bluesky using Qt6?

Reply:
That is a site-level choice, in https://github.com/bluesky/bluesky/pull/1512 we got it to work and had a test, but I'm not sure if the pyqt6 dep survived a testing refactor...